Source/WebKit/WPEPlatform/wpe/WPESettings.h

```diff
@@ -195,6 +195,58 @@ typedef enum {
  * Default: 400
  */
 #define WPE_SETTING_DOUBLE_CLICK_TIME "/wpe-platform/events/double-click/time"
+/**
+ * WPE_SETTING_CREATE_TOPLEVEL_ON_VIEW_CREATION:
+ *
+ * If a new toplevel is created upon creation of a new #WPEView.
+ *
+ * VariantType: boolean
+ *
+ * Default: true
+ */
+#define WPE_SETTING_CREATE_TOPLEVEL_ON_VIEW_CREATION "/wpe-platform/create-toplevel-on-view-creation"
```

Comment:
I would leave this for a separate commit, since the feature hasn't been implemented yet.